.github/workflows/tests-unit-php.yml

```
@@ -113,10 +103,13 @@ jobs:
           coverage: ${{ matrix.coverage && 'xdebug' || 'none' }}
           tools: composer, cs2pr
 
-      - name: Install dependencies
-        run: |
-          composer install --prefer-dist --no-progress --no-interaction --no-scripts
-          composer dump-autoload --no-interaction
+      - name: Install PHP dependencies
+        uses: "ramsey/composer-install@v1"
+        with:
+          composer-options: "--prefer-dist --no-progress --no-interaction --no-scripts"
+
+      - name: "Composer dump autoload"
```

```suggestion
      - name: Composer dump autoload
```

No other job step names have quotes